common/src/main/java/com/microsoft/identity/common/internal/logging/Logger.java

```diff
@@ -64,129 +62,108 @@
     }
 
     public static void setAllowPii(final boolean allowPii) {
-        logDeprecationWarning();
         com.microsoft.identity.common.logging.Logger.setAllowPii(allowPii);
     }
 
     public static void setAllowLogcat(final boolean allowLogcat) {
-        logDeprecationWarning();
         com.microsoft.identity.common.logging.Logger.setAllowLogcat(allowLogcat);
     }
 
     public static boolean getAllowPii() {
-        logDeprecationWarning();
         return com.microsoft.identity.common.logging.Logger.getAllowPii();
     }
 
     public static boolean getAllowLogcat() {
-        logDeprecationWarning();
         return com.microsoft.identity.common.logging.Logger.getAllowLogcat();
     }
 
     public static String getDiagnosticContextMetadata() {
-        logDeprecationWarning();
```

Comment:
Hm.  Should we add a Telemetry.emit() in the place of the logger?  Eventually we'd like to be able to have things like this in metrics. @shahzaibj for thoughts.

Reply:
Yes, let add telemetry. Let's create a new `Event` called something like `DeprecatedApiUsageEvent` (or some similar name), and `emit` that event whenever a deprecated api is used. The event should capture the class and name of the API that was used.